package/Maps/Yuri's Revenge/4_treetop_hills.map

```diff
@@ -0,0 +1,3696 @@
+; Map created with FinalAlert 2(tm) Mission Editor
+; Get it at http://www.westwood.com
+; note that all comments were truncated
+
+[Header]
+Width=95
+Height=90
+StartX=208
+StartY=59
+Waypoint1=247,66
+Waypoint2=215,87
+Waypoint3=245,130
+Waypoint4=289,101
+Waypoint5=0,0
+Waypoint6=0,0
+Waypoint7=0,0
+Waypoint8=0,0
+NumberStartingPoints=4
+
+[GAPOWR]
+Sight=7
+BuildTimeMultiplier=0.75
```

Comment:
hard coded allied power plant buff?

Reply:
Yes, he used this code to make it easier to build on some maps.

Reply:
let's not hardcode the power plant buff into maps